mantidproject/fitbenchmarking
# Deleted unused code

#### Description of Work
Fixes #33 
Mainly got rid of unused code and made small cosmetic changes to the code.

#### Test Instructions
1. In `example_runScript.py`, set `run_data = "neutron"`.
2. Run `example_runScript.py` in the mantidpython console.
3. Check if the output changed with respect to the output from master.
4. Repeat with `run_data = "nist"`.
